src/IssueViz.Security/OpenInIDE/Http/OwinPipelineProcessor.cs

```diff
@@ -54,6 +55,7 @@ public OwinPipelineProcessor([ImportMany]IEnumerable<IOwinPathRequestHandler> pa
 
             foreach(var handler in pathRequestHandlers)
             {
+                Debug.Assert(handler.ApiPath.StartsWith("/"), "Expecting the handler path to start with /");
```

Comment:
This is to fix an issue I found when manually testing; the "show" request wasn't being passed to the handler class.

The Owin HttpListener is listening to all request on `http://localhost:[port]/sonarlint/api/` and it strips off that prefix before forwarding the request to the next handler in the pipeline. However, the `context.Request.Path` value it forwards always starts with '/'.

Originally, I had the HttpListener configured to listen to all requests on `localhost:[port]` - it was a late change to make it listen only to `sonarlint/api/`, and I can't have restarted manually in the IDE...